src/waveform/renderers/waveformrendermark.cpp

```diff
@@ -31,7 +31,8 @@ void WaveformRenderMark::setup(const QDomNode& node, const SkinContext& context)
 
 void WaveformRenderMark::draw(QPainter* painter, QPaintEvent* /*event*/) {
     PainterScope PainterScope(painter);
-
+    // Maps mark image offset to mark object.
+    QMap<WaveformMarkPointer, int> marksOnScreen;
```

Comment:
This comment wouldn't be necessary with a better name for the variable. How about `marksToScreenPositions`?

Reply:
Renaming this variable should be better. But the positions are relative to the scrolling widget, not the screen. So it may be renamed to `marksToWidgetPositions`. But the word `screen` indicates that this `QMap` only stores marks visible on screen.

In the overview widget, the size of the mark image is equal to the widget, thus, this position information is not required to detect if a mouse pointer is near/on the marker.
But I wouldn't suggest using the complete widget area just to draw markers on the scrolling waveform since that would take >10x canvas space required. So we will keep sending position with marks as a `QMap`.

Reply:
Okay, then change the comment to
```suggestion
    // Maps mark object to its position in the widget
    QMap<WaveformMarkPointer, int> marksOnScreen;
```